demos/http/http_demo_s3_download_multithreaded/http_demo_s3_download_multithreaded.c

```diff
@@ -835,200 +840,195 @@ int main( int argc,
     /* Status to the host OS indicating a successful demo or not. */
     int32_t returnStatus = EXIT_SUCCESS;
 
-    ( void ) argc;
```

Comment:
Does taking this not cause warnings?

Reply:
It was moved down, since the variables that were at top of the for loops scope were moved above it